addons/wiki/static/dragNDrop.js

```diff
@@ -9,6 +9,11 @@ var getExtension = function(filename) {
 var validImgExtensions = ['jpg', 'jpeg', 'png', 'gif', 'bmp'];
 var imageFolder = 'Wiki images';
 
+// Stop people from dropping files in the gutter while editor is disabled.
+$('.ace_gutter').bind("dragover", function(e) {
```

Comment:
This is OK, but would it possible to just bind the correct element (i.e. the textarea, excluding the gutter) to the `dragover` listener here: https://github.com/CenterForOpenScience/osf.io/blob/d61e52882ca1801e4f94088cd4f7f761af92262f/addons/wiki/static/dragNDrop.js#L185 ?